### PR TITLE
Updating AST to 5.5 and adding in players hit to div for arcana tracking

### PR DESCRIPTION
**Pull request by otocephaly**
- Updated AST to 5.5 support and updated changelog
- Added myself to contributors
- Updated Arcana Tracking to include Players Buffed by Divination
  - The code utilized was largely taken from DRG Battle Litany code with some tweaks to fit the Arcana Tracking table
  - See attached screenshot for sample tested

Please let me know if you have any questions or concerns about this PR
Thank you

DivTest

### CODE REVIEW

File: src/parser/jobs/ast/index.tsx
Comment (suShirow) [inline src/parser/jobs/ast/index.tsx:33]: I'm pretty sure Earthly star is still bugging out a little at this point, I was halfway through a fix before I had to go on hiatus. But regardless I agree it's fine to bump since there really were no changes since like ~5.3

**Review by suShirow — APPROVED** [review]
Logic checks out! Thanks so much for contributing and bringing life back into this side of xiva. 
Sorry but before we can merge, we will need the status module to use the new patch system, same like actions.

File: src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx
Comment (otocephaly) [reply src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx]: hi sushi! I used this.data.statuses as advised in the new commit
Please let me know if you have any further feedback
thank you

File: src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx
Comment (otocephaly) [reply src/parser/jobs/ast/modules/ArcanaTracking/ArcanaSuggestions.tsx]: hi sushi
I have incorporated your feedback in the new commit
please let me know if you have any further feedback
Thanks!

File: src/parser/jobs/ast/index.tsx
Comment (otocephaly) [reply src/parser/jobs/ast/index.tsx:33]: good to know!
please let me know if you'd like help with that earthly star fix

**Review by suShirow — APPROVED** [review]

**Comment by suShirow** [comment]
I'll merge it when I get home tonight.

**Comment by ayyaruq** [comment]
Quick note, merging since Sushi approved, but the DIVWindow class here would have been better off as an interface since the class has no methods, it's just structural. The main reason to use classes is to have getters or other methods on them to work with the data but if it's just for the field names, an interface will suffice.

**Pull request merged**
